libzotero/libzotero.py

```diff
@@ -26,6 +26,33 @@
 import time
 from libzotero.zotero_item import zoteroItem as zotero_item
 
+def decode_zotero_str(raw, max_iters=3):
```

Comment:
I'm on Linux. I have that fork running on 3 different systems,
- two of which are Ubuntu 14.04 based Elementary OS Freya
- and one is Ubuntu 16.04 based Elementary OS Loki.